lib/sms.rb

```diff
@@ -0,0 +1,16 @@
+class Sms
+
+  def initialize
+    @tokens = YAML::load(File.open('./lib/resources/tokens.yml'))
```

Comment:
Style/ColonMethodCall: Do not use `::` for method calls. (https://github.com/bbatsov/ruby-style-guide#double-colons)